specification/communication/data-plane/Messages/readme.md

```diff
@@ -40,13 +51,7 @@ title:
   Azure Communication Services
 ```
 
-# Code Generation
-
-## Swagger to SDK
-
-This section describes what SDK should be generated by the automatic system.
-This is not used by Autorest itself.
-
-## CSharp
-
-See configuration in [readme.csharp.md](./readme.csharp.md)
+``` yaml
+suppressions:
+  - code: LroExtension
```

Comment:
What is this suppression for?

Reply:
It is here to suppress "Operations with a 202 response must specify x-ms-long-running-operation: true." It is advised by Laurent Mazuel to add this suppression. It is safe to ignore as the service will send the long-running response through EG events.